common/src/main/java/com/github/khanshoaib3/minecraft_access/utils/condition/MenuKeystroke.java

```diff
@@ -24,6 +24,9 @@ public MenuKeystroke(BooleanSupplier condition) {
         super(condition);
     }
 
+    /**
+     * Pure function
+     */
```

Comment:
Consider using JetBrains' `@Contract` annotation to get warnings for incorrect usage.
```suggestion
    @Contract(pure = true)
```